app/components/slack-context-menu.js

```diff
@@ -138,7 +149,7 @@
 	}
 
 	module.exports = {
-		handleRightClick: function (window, target, x, y) {
+		handleRightClick: function (window, target, x, y, teamUrl) {
```

Comment:
I felt the same (as pointed out in my initial message) but didn't want to change the approach without prior discussion. Eventually, the `team` prop is something that could be interesting to have as React context instead!

I can change the arguments to use a parameters object instead.